examples/spark_udf/complex.py

```diff
@@ -0,0 +1,92 @@
+import random
```

Comment:
@harupy Nit: Can we adjust the name of the example? Not sure if we want to tell users that our example is "complex"

Reply:
Renamed!